bigbluebutton-html5/private/config/settings.yml

```diff
@@ -380,7 +380,7 @@ public:
   poll:
     enabled: true
     max_custom: 5
-    allowDragAndDropFile: false
+    allowDragAndDropFile: true
```

Comment:
Hey @germanocaumo , is this change intentional?

Reply:
Oops no, sorry, reverted.